src/SignalRClient/Program.cs

```diff
@@ -63,6 +64,20 @@ static async Task Main(string[] args)
             {
                 Console.WriteLine("SignalR Client");
 
+                Console.WriteLine("#StartJobStatistics");
```

Comment:
Funny ;) There are two way to expose a metric, either using the event source, or using the console. I know I pointed you to a console example ... I was not clear. But you are already using the event source in the rest of the code, so i think you should do it there too, nicer.

```
BenchmarksEventSource.Log.Metadata("signalr/raw-errors", "all", "all", "Raw errors", "Raw errors", "object");
```